kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/steps/KafkaSteps.java

```diff
@@ -38,6 +42,29 @@ public static void createTopic(String clusterName, String topicName, String name
                 KafkaTopicTemplates.defaultTopic(namespace, clusterName, topicName, partitions, replicas, minIsr).build());
     }
 
+    /**
+     * Create topic using test clients.
+     *
+     * @param topicName the topic name
+     * @param bootstrap the bootstrap
+     * @param partitions the partitions
+     * @param replicas the replicas
+     */
+    public static void createTopicTestClient(String deployNamespace, String topicName, String bootstrap, int partitions, int replicas) {
+        String podName = KafkaUtils.AdminTestClient(deployNamespace, bootstrap);
+        String command = "admin-client topic create --bootstrap-server=" + bootstrap + " --topic=" + topicName + " --topic-partitions=" + partitions +
+                " --topic-rep-factor=" + replicas;
+
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
```

Comment:
you passing a stream but then not doing anything with it.  You probably want to log want the contents if the result is non-zero.

Reply:
done. Thanks!